src/evidently/metrics/regression_performance/mae.py

```diff
@@ -0,0 +1,74 @@
+from typing import List
+from typing import Optional
+
+import numpy as np
+from sklearn.metrics import mean_absolute_error
+
+from evidently.base_metric import InputData
+from evidently.base_metric import Metric
+from evidently.base_metric import MetricResult
+from evidently.metric_results import SimpleMetricValues
+from evidently.model.widget import BaseWidgetInfo
+from evidently.renderers.base_renderer import MetricRenderer
+from evidently.renderers.base_renderer import default_renderer
+from evidently.renderers.html_widgets import CounterData
+from evidently.renderers.html_widgets import counter
+from evidently.renderers.html_widgets import header_text
+
+
+class MAEMetricResults(MetricResult):
+    current: SimpleMetricValues
+    reference: Optional[SimpleMetricValues]
+
+
+class MAEMetric(Metric[MAEMetricResults]):
```

Comment:
If we plan to do more of those, lets also generalize. Eg `class SimpleMetric(Metric[SimpleMetricResults])`
Maybe even renderer can be the same